examples/mlx_metal_kernel_opt/test_optimized_attention.py

```
@@ -0,0 +1,498 @@
+    python test_optimized_attention.py [path_to_best_program.py]
+
+    If no path is provided, it will use the default best_program.py from
+    openevolve_output/best/
+"""
+
+import os
+import sys
+import time
+import argparse
+import subprocess
+import tempfile
```

Ruff (0.11.9)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)